src/pages/graphql/schema/customer/queries/customer.md

```diff
@@ -716,6 +716,164 @@ The following query returns the customer's wish lists. Adobe Commerce allows cus
 }
 ```
 
+### List a company user's purchase orders
+
+The following example returns information about all purchase orders the company user has made since January 1, 2023.
+
+[Purchase orders](../../b2b/purchase-order/) contains additional examples that query purchase order functionality.
```

Comment:
I see you are referring to linked pages as singular even when their linked text is plural. I'm fine with it, if this is the standard, just pointing it out.